### PR TITLE
Hy/preferences all toggles enabled

### PR DESCRIPTION
**Pull request by sv-hyacoub**
# Description

This test ensures that sponsored/nonsponsored fx suggestion are toggled on in about:preferences#search, and strings introduced in the address bar can trigger sponsored/nonsponsored suggestions.

## Bugzilla bug ID

**ID: 1902782**
**Link: https://bugzilla.mozilla.org/show_bug.cgi?id=1902782**

## Type of change

Please delete options that are not relevant.

- [x] New Test

# How does this resolve / make progress on that bug?

Completed.

# Screenshots / Explanations

N/A.

# Comments / Concerns

Please add a short blurb about any comments or concerns that this change might cause.

### CODE REVIEW

File: tests/address_bar_and_search/test_preferences_all_toggles_enabled.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+@pytest.fixture()
+def add_prefs():
+    return [
+        ("browser.search.region", "US"),
+    ]
+
+
+def test_preferences_all_toggles_enabled(driver: Firefox):
```
Comment (sarinali) [inline tests/address_bar_and_search/test_preferences_all_toggles_enabled.py:18]: I think adding the `@pytest.mark.slow` is appropriate, also maybe you can write a one line comment documenting why you have the long sleeps?

**Review by sarinali — COMMENTED** [review]
minor suggestion, otherwise looks good

File: tests/address_bar_and_search/test_preferences_all_toggles_enabled.py (hunk 1 above)
Comment (sv-hyacoub) [reply tests/address_bar_and_search/test_preferences_all_toggles_enabled.py:19]: Thanks for the review, I just added the suggestions.

**Review by sarinali — APPROVED** [review]
LGTM!

**Pull request merged**